service/src/test/resources/application-azure-test.yml

```diff
@@ -0,0 +1,20 @@
+workspace:
+  azure-test:
+    # tenant: DSP Terra Test User 2
+    # tenant-id: 5571f741-d752-48d0-b191-1922a1f0c755
+
+    # tenant: broadinstitute.org
+    tenant-id: 0cb7a640-45a2-4ed6-be9f-63519f86e04b
+
+    # subscription: 8201533_TDR_testuser2
+    # subscription-id: 19a55c49-f0c9-4149-bb96-5c08781d0cdf
+
+    # subscription: 8201533_Interactive Analysis Playground
+    subscription-id: 3efc5bdf-be0e-44e7-b1d7-c08931e3c16c
+
+    # MRG
+    managed-resource-group-id: mrg-qi-20210930145737
```

Comment:
Can we use `mrg-terra-integration-test-20211118`?

It doesn't really matter which MRG we use, but I set up that MRG for the purpose of integration testing. Janitor uses it too: https://github.com/DataBiosphere/terra-resource-janitor/blob/master/src/test/resources/application-integration.yml#L22

Reply:
Yes, of course. Seems better than using  @Qi77Qi's!